src/main/java/duke/ToDos.java

```diff
@@ -0,0 +1,17 @@
+package duke;
+
+import duke.Task;
+
+public class ToDos extends Task {
```

Comment:
Suggest to change class name to Todo, since each instance of this class represents a single Todo.